cli/README.md

```diff
@@ -123,6 +123,7 @@ cd /usr/share/FreshRSS
 
 ./cli/db-backup.php
 # Back-up all users respective database to `data/users/*/backup.sqlite`
+# -q, --quiet suppress all normal output
```

Comment:
```suggestion
# -q, --quiet suppress progress information
```

Or something like that

Reply:
Indeed, it is difficult to convey the meaning in this case.
"suppress non-error messages" in the rsync help document seems clear to me, what do you think?
Sure, "progress information" whould be good, too.
Messages such as "Start SQL copy..." can also be categorized as it in a broad meaning.

For reference, the help documentation for `--quiet` option of major commands is below:

- `rsync`: suppress non-error messages
- `gzip`: suppress all warnings
- `git clone`: Operate quietly. Progress is not reported to the standard error stream.